src/components/global/footer.tsx

```diff
@@ -251,8 +271,33 @@ const StyledSlideText = styled.span`
   }
 `;
 
+const StyledPillIconButton = styled(PillIconButton)`
+
+  &:hover .styled-icon {
+    transform: rotate(0deg);
+  }
+  &:hover {
+    path {
+      stroke: ${colors.white};
+      fill: none;
+    }
```

Comment:
Closing final bracket

```suggestion
  &:hover {
    path {
      stroke: ${colors.white};
      fill: none;
    }
  }
```